src/tests/ftest/util/dmg_utils.py

```diff
@@ -30,6 +30,10 @@
 from command_utils import FormattedParameter
 
 
+class DmgFailure(Exception):
+    """Base exception for this module."""
+
+
```

Comment:
Is this required over `CommandFailure`? Where is this exception being raised?

Reply:
There is none. I'll remove it.